### DISCUSSION TITLE
[BUG] Windows Agent Log File Huge

### DISCUSSION THREAD

**Discussion by amanzella** [utmstack/UTMStack#580] [Q&A] [answered]
We've noticed that the log file beats_windows_agent under C:\Program Files\UTMStack\UTMStack Agent\logs_process is huge. It's currently 25GB in size. Does the agent not clean up its logs?

**Comment by c3s4rfred** [comment]
Hi @amanzella, can you send us your agent version, send us the content of this file -> C:\Program Files\UTMStack\UTMStack Agent\versions.json

**Comment by amanzella** [comment]
@c3s4rfred 

`{"testing": {"beat_version": "8.5.3", "windows_dependencies": "agent_updates/testing/windows/dependencies.zip", "linux_dependencies": "agent_updates/testing/linux/dependencies.zip", "agent_version": "202311020758", "windows_agent": "agent_updates/testing/windows/utmstack-windows.exe", "linux_agent": "agent_updates/testing/linux/utmstack-linux"}, "alpha": {"beat_version": "8.5.3", "windows_dependencies": "agent_updates/alpha/windows/dependencies.zip", "linux_dependencies": "agent_updates/alpha/linux/dependencies.zip", "agent_version": "202311020758", "windows_agent": "agent_updates/alpha/windows/utmstack-windows.exe", "linux_agent": "agent_updates/alpha/linux/utmstack-linux"}, "release": {"beat_version": "8.5.3", "windows_dependencies": "agent_updates/release/windows/dependencies.zip", "linux_dependencies": "agent_updates/release/linux/dependencies.zip", "agent_version": "10.1.1", "windows_agent": "agent_updates/release/windows/utmstack-windows.exe", "linux_agent": "agent_updates/release/linux/utmstack-linux"}}`

**Comment by c3s4rfred** [comment, marked as answer]
@amanzella you're using an old UTMStack version, that issue was fixed in later versions, please install the latest version and let us know if solved.

**Reply by amanzella** [reply]
@c3s4rfred What's the proper process to update the client?

**Reply by c3s4rfred** [reply]
You only have to remove the old UTMStack installer (not the agent installer), download the new one, and execute it as you do the first time. The agent should update automatically.
